scripts/release/publish-docs-artifacts.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Publish material2 docs assets to the material2-docs-content repo
+# material.angular.io will pull from this assets repo to get the latest docs
+
+cd "$(dirname $0)/../../"
+root="$(pwd)"
+
+docsPath="./dist/docs"
+tempRepoPath="./dist/tempRepo"
+repoUrl="https://github.com/angular/material2-docs-content/"
+
+# If the docs directory is not present, abort
+if [ ! -d $docsPath ]; then
+  echo "Material docs not generated. Aborting..."
+  exit 0
+fi
+
+# Get git meta info for commit
+commitSha="$(git rev-parse --short HEAD)"
+commitAuthorName="$(git --no-pager show -s --format='%an' HEAD)"
+commitAuthorEmail="$(git --no-pager show -s --format='%ae' HEAD)"
+commitMessage="$(git log --oneline -n 1)"
+
+# create directory and clone test repo
+rm -rf $tempRepoPath
+mkdir $tempRepoPath
+git clone $repoUrl $tempRepoPath
+
+# Clean out repo directory and copy contents of dist/docs into it
+$(find $tempRepoPath -d 1 | grep -v "git" | xargs rm -rf)
+mkdir $tempRepoPath/docs
+cp -r $docsPath/* $tempRepoPath/docs
+
+# Push content to repo
+cd $tempRepoPath
+# git remote show origin
+git config user.name "$commitAuthorName"
+git config user.email "$commitAuthorEmail"
+git config credential.helper "store --file=.git/credentials"
```

Comment:
Since the Github credentials are now retrieved from the `.git/credentials` file, the credentials file needs to be created.

```bash
echo "https://${MATERIAL2_BUILDS_TOKEN}:@github.com" > .git/credentials
```